shops/client/src/components/ProductsList.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+import { Product } from "../Product";
+import { getProducts } from "../../../services/productService";
+import ProductCard from "../../src/components/ProductCard";
+
+const ProductList = () => {
+  const [products, setProducts] = useState<Product[]>([]);
+
+  useEffect(() => {
+    const fetchProducts = async () => {
```

Comment:
I would replace lines 10-19 with the function you made in your service folder. Then just assign the response data to your products state. 

```    getProducts().then((productData) => setProducts(productData));```